### DISCUSSION TITLE
Support for multiple OpenAPI specifications

### DISCUSSION THREAD

**Discussion by obfischer** [docToolchain/docToolchain#1167] [Q&A]
Dear all,

our project consists of many independent OpenAPI specifications, we would like to export with the task `exportOpenAPI` to AsciiDoc. But the task fails, if I add the `openAPI` block more then one time.

Any idea how I can solve this issue?

**Comment by mh182** [comment]
Thanks for requesting this feature.

At the moment, the `exportOpenApi` task doesn't support the export of multiple OpenAPI specifications .

The workaround for the moment would be to change the configuration to a specific OpenAPI spec, calling `exportOpenApi` and add the newly generated Asciidoc file of the OpenAPI to your `src/docs` folder in your git repo. Then proceed to the next one.

Whenever you change the OpenAPI spec, you would have to re-export the Asciidoc file again.

Maybe you could take a look into https://github.com/docToolchain/docToolchain/blob/ng/scripts/exportOpenApi.gradle and implement it on your own. A PR is always welcome.

**Reply by mh182** [reply]
I created a feature request  #1171 from your question here.
